exampleSite/config.toml

```diff
@@ -143,35 +140,35 @@ paginate = 10
     enable = true
     # All carousel items are defined in their own files. You can find example items
     # at 'exampleSite/data/carousel'.
-    # For more informtion take a look at the README.
+    # For more information take a look at the README.
```

Comment:
@varnerac  I would appreciate misspelling words to be in another PR directly to master branch.